### PR TITLE
Add show-sbom finally task

### PR DESCRIPTION
**Pull request by mmorhun**
This PR adds a new task `show-sbom` in finally section that prints SBOM in CycloneDX format.
The task is added into all build pipelines.
If the build failed - the task will not be run.

Issue: https://issues.redhat.com/browse/STONEBLD-1042

image

### CODE REVIEW

File: task/show-sbom/OWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+RHTAP Build Team
```
[the remark this answers is not in the document]
Comment (mmorhun) [reply task/show-sbom/OWNERS:1]: I see, thanks

**Comment by mmorhun** [comment]
Rebased

**Review by psturc — COMMENTED** [review]
would be good to have e2e passed on this PR, cc @Michkov 

otherwise lgtm

**Review by tkdchen — APPROVED** [review]

File: task/show-sbom/0.1/show-sbom.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: show-sbom
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "appstudio, hacbs"
```
Comment (tkdchen) [inline task/show-sbom/0.1/show-sbom.yaml:10]: Still use hacbs?

Comment (mmorhun) [reply task/show-sbom/0.1/show-sbom.yaml:10]: It's present in all labels. I think it should be a separate PR that deletes HACBS stuff (and not only from label)

**Review by Michkov — APPROVED** [review]

**Pull request merged**